Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian:stretch
+
+ENV LANG=C.UTF-8
+
+RUN set -x \
+  && apt-get update \
+  && apt-get install -y \
+    build-essential \
+    libacl1 \
+    libacl1-dev \
+    libssl-dev \
+    liblz4-dev \
+    liblz4-1 \
+    python3-pip \
+  && pip3 install borgbackup \
+  && apt-get remove -y --purge build-essential libssl-dev liblz4-dev libacl1-dev \
+  && apt-get autoremove -y --purge \
+  && rm -rf /var/lib/apt/lists/* \
```

Comment:
do apt clean (first)?

Reply:
You could/should just use a 2 stage container build instead (one for the build dependencies, one with only runtime dependencies and the output of the build).